src/Domain/SettingsPage.php

```diff
@@ -0,0 +1,346 @@
+<?php
+namespace GiveFBPT\Domain;
+
+/**
+ * Example code to show how to add setting page to give settings.
+ *
+ * @package     GiveFBPT\Addon
+ * @subpackage  Classes/Give_BP_Admin_Settings
+ * @copyright   Copyright (c) 2020, GiveWP
+ */
+class SettingsPage extends \Give_Settings_Page {
```

Comment:
Make sure to clean this up as well.

Reply:
See this comment: https://github.com/impress-org/givewp-facebook-pixel-tracking/pull/8#discussion_r506598069